templates/_images/fedora-42.yaml

```diff
@@ -7,6 +7,9 @@ images:
   arch: aarch64
   digest: sha256:e10658419a8d50231037dc781c3155aa94180a8c7a74e5cac2a6b09eaa9342b7
 
+- location: https://dl.fedoraproject.org/pub/alt/risc-v/release/42/Cloud/riscv64/images/Fedora-Cloud-Base-Generic-42.20250414-8635a3a5bfcd.riscv64.qcow2
```

Comment:
Do you wan to update fedora-41.yaml too?

Also wonder if we can cover alt/risc-v in https://github.com/lima-vm/lima/blob/master/hack/update-template-fedora.sh

Reply:
> Do you wan to update fedora-41.yaml too?

I didn't realise we had 41. Yep, I'll test that it boots and add it.

> Also wonder if we can cover alt/risc-v in https://github.com/lima-vm/lima/blob/master/hack/update-template-fedora.sh

I spent a few minutes trying to hack it in, but I didn't get it working. I'm not as much of a shell wizard as rhe author of those scripts!

Reply:
cc @norio-nomura